src/seedu/addressbook/Main.java

```diff
@@ -16,10 +16,18 @@
     /** Version info of the program. */
     public static final String VERSION = "ClassRepo - Version 1.1";
 
+    public static void main(String[] args) {
+        launch(args);
+    }
+
     @Override
-    public void start(Stage primaryStage) throws Exception {
-        Gui gui = new Gui(new Logic(), VERSION);
-        gui.start(primaryStage, this);
+    public void start(Stage primaryStage) {
+        try {
+            Gui gui = new Gui(new Logic(), VERSION);
+            gui.start(primaryStage, this);
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
```

Comment:
So here, you will catch the exception and throw it again

Reply:
i added a runtimeexception for this as requested

Reply:
Changed to Exception

Reply:
Changed to IOException and passed codacy

Reply:
Erg, now its casting it into a new type of exception.
It should just be throw e. No need to cast it into a new type of exception. This is to keep the category the original exception is part of (StorageIO and other exceptions), But  if ya want to merge it still, then I'll just fix it in the next patch